src/main/java/core/basesyntax/model/abstraction/DrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.model.abstraction;
+
+public interface DrawFigure {
```

Comment:
Interface - is an abstraction. It gives the possibility to draw. It gives it to Figure, to Plane, to Car, to Cat... you name it.
Interface must know NOTHING about its implementations. It is one of the cornerstones of OOP and SOLID principles.
Delete 'Figure' from the interface.